### ISSUE TITLE
User data exceeds 16kb

### ISSUE THREAD

**Issue by razor54** [philips-labs/terraform-aws-github-runner#3700]
```╷
│ Error: creating EC2 Launch Template (lt-0d19cc63e67d12cbf) Version: InvalidUserData.Malformed: User data is limited to 16384 bytes.
│ 	status code: 400, request id: d433057d-2bfb-4ce5-a121-26ad9048dbd2
│
│   with module.runners.module.runners.aws_launch_template.runner,
│   on .terraform/modules/runners/modules/runners/main.tf line 64, in resource "aws_launch_template" "runner":
│   64: resource "aws_launch_template" "runner" {
│
╵
```
I used to be able to run ~6.8kb of custom init scripts, but now I get the above error after updating from `5.3.0` to `5.6.0`

```
-rw-r--r-- 1 andregaudencio staff 3.1K Feb 11  2023 post-install.sh
-rw-r--r-- 1 andregaudencio staff 3.7K Jan  2 11:50 user-data.sh
```

whilst the final base64 encoded string appears to be ~44kb, 
```
-rw-r--r-- 1 andregaudencio staff 44K Jan  2 11:47 base64-string
```

**Comment by razor54** [comment]
Also tested on `5.4.2` and got the same result. Reverted to `5.3.0`

**Comment by npalm** [comment]
Please can anyone try the fix in PR #3717

**Comment by razor54** [comment]
That worked for me :)
